cloudformation/membership-attribute-service.yaml

```diff
@@ -104,12 +93,14 @@ Resources:
       - PolicyName: root
         PolicyDocument:
           Statement:
-          - Action: s3:GetObject
-            Resource: arn:aws:s3:::gu-membership-*/*
-            Effect: Allow
-          - Action: s3:GetObject
-            Resource: arn:aws:s3:::gu-reader-revenue-private/${Stack}/members-data-api/${Stage}/*
```

Comment:
I think this is actually broken (missing a !Sub), but we never actually noticed because AmazonEC2RoleforSSM provides access to the bucket.